packages/yew-router/src/lib.rs

```diff
@@ -88,6 +88,10 @@ pub mod history {
         AnyHistory, BrowserHistory, HashHistory, History, HistoryError, HistoryResult, Location,
         MemoryHistory,
     };
+
+    pub(crate) mod query {
+        pub(crate) use gloo::history::query::Raw;
+    }
```

Comment:
Why not just use `gloo::history::query::Raw` directly? I don't see the point of this re-export

Reply:
Seemed more consistent, but I will revise according to your suggestion!